native/.idea/codeStyles/Project.xml

```diff
@@ -9,7 +9,7 @@
       <option name="SPACE_BEFORE_GENERATOR_MULT" value="true" />
       <option name="USE_DOUBLE_QUOTES" value="false" />
       <option name="FORCE_QUOTE_STYlE" value="true" />
-      <option name="OBJECT_LITERAL_WRAP" value="2" />
+      <option name="OBJECT_LITERAL_WRAP" value="0" />
```

Comment:
I think the option we want is `chop down if to long`:
```suggestion
      <option name="OBJECT_LITERAL_WRAP" value="5" />
```
This option keeps all params in one line if shorter than max line length and puts each param in a separate line else. Other opinions?
Please verify if this does what I think it does and whether you like it.